cli/cmd/component.go

```diff
@@ -1215,3 +1181,41 @@ func downloadProgress(complete chan int8, path string, sizeB int64) {
 		time.Sleep(time.Second)
 	}
 }
+
+func LoadCatalog() (*lwcomponent.Catalog, error) {
+	cli.StartProgress("Loading component Catalog...")
+	defer cli.StopProgress()
+
+	var componentsApiInfo map[string]lwcomponent.ApiInfo
+
+	// try to load components Catalog from cache
+	if !cli.noCache {
+		expired := cli.ReadCachedAsset(componentsCacheKey, &componentsApiInfo)
+		if !expired {
+			cli.Log.Infow("loaded components from cache", "components", componentsApiInfo)
+			catalog, err := lwcomponent.NewCachedCatalog(cli.LwApi, lwcomponent.NewStageTarGz, componentsApiInfo)
+			if err != nil {
+				return nil, err
+			}
+			return catalog, nil
```

Comment:
no need to check for errors
```suggestion
			return lwcomponent.NewCachedCatalog(cli.LwApi, lwcomponent.NewStageTarGz, componentsApiInfo)
```